### PR TITLE
remove okteto cloud entry in okteto context list

### PR DESCRIPTION
**Pull request by AdrianPedriza**
### Changes
- remove okteto cloud entry as part of okteto context list

### CODE REVIEW

File: src/content/reference/cli.mdx [hunk 1]
```diff
@@ -128,7 +128,6 @@ This will prompt you to select one of your existing contexts or to create a new
 A context defines the default cluster/namespace for any Okteto CLI command.
 Select the context you want to use:
 Use the arrow keys to navigate: ↓ ↑ → ←
-    https://cloud.okteto.com (Okteto Cloud)
     minikube
   ▸ https://okteto.example.com *
```
[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply src/content/reference/cli.mdx:132]: We currently do not follow any order for this, regardless of the context in use. We were only showing `https://cloud.okteto.com (Okteto Cloud)` always the first one. If you think that aesthetically it is better in the documentation I can change it but the CLI does not keep order in the displayed output

[the remark this answers is not in the document]
Comment (pchico83) [reply src/content/reference/cli.mdx:132]: I think we always show the okteto contexts first, and then the kubernetes contexts

[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply src/content/reference/cli.mdx:132]: yes, that's right. First we show the okteto contexts. I update the PR

[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply src/content/reference/cli.mdx:131]: 0a64655d1fa1f372d39547c6ffe9a74ad0a76578

**Comment by omnomagonz** [comment]
Jumping in to confirm: is this PR good to merge? Catching up on notifications 😄

**Comment by AdrianPedriza** [comment]
> Jumping in to confirm: is this PR good to merge? Catching up on notifications 😄

I think so, I merge it

**Pull request merged**